src/inputs/TextFieldBase.tsx

```diff
@@ -143,6 +149,12 @@ export function TextFieldBase<X extends Only<TextFieldXss, X>>(props: TextFieldB
         : Css.bcGray300.if(contrast).bcGray700.$),
       // Do not add borders to compound fields. A compound field is responsible for drawing its own borders
       ...(!compound ? Css.ba.$ : {}),
+      ...(borderOnHover && Css.pl1.ml(-1).br4.ba.bcTransparent.$),
+      ...(borderOnHover && Css.if(isHovered).bgBlue100.ba.bcBlue300.$),
+      ...{
+        // Highlight the field when hovering over the row in a table
+        [`.${ROW_CSS_SELECTOR}:hover:not(:has(.textFieldBaseWrapper:hover)) &`]: Css.ba.bcBlue300.$,
```

Comment:
What do we need this `not(has(...))` for? Afaict if I remove the `not/has`, everything still works the same?

Reply:
Oh, this is saying "if one text field cell is hovered, remove the outline from the other text field cells"?

Reply:
Yup, exactly